### PR TITLE
Target libraries - fix penumbra dns

### PR DESCRIPTION
**Pull request by agouin**
Penumbra `pclientd` binary requires `libnss_dns.so.2` and `libresolv.so.2` in order to resolve DNS.

Adds `target-libraries` to config for copying libraries from the target image.

### CODE REVIEW

File: addChain.md [hunk 1]
```diff
@@ -23,9 +23,11 @@ Please keep chains in alphabetical order.
 
 `build-target` -> The build command specific to the chosen `dockerfile`. For `cosmos`, likely `make install`. For `cargo`, likely `build --release`.
 
-`binaries` -> The location of where the the build target places the binarie(s). Adding a ":" after the path allows for the ability to rename the binary.
+`binaries` -> The location of the binary(ies) in the build environment after the build is complete. Adding a ":" after the path allows for the ability to rename the binary.
 
-`libraries` -> Any extra libraries need to run the binary. In additon to the binary itself, these will be copied over to the final image
+`libraries` -> Any extra libraries from the build environment needed in the final image.
+
+`target-libraries` -> Any extra libraries from the target image needed in the final image.
```
Comment (DavidNix) [inline addChain.md:30]: I'm probably missing something obvious, but why can't we reuse `libraries`?

Comment (DavidNix) [reply addChain.md:30]: Also, this setting seems to only affect cargo docker files. Should we specify that in the docs here?

Comment (agouin) [reply addChain.md]: > Also, this setting seems to only affect cargo docker files. Should we specify that in the docs here?

Good catch, added note

Comment (DavidNix) [reply addChain.md]: Thanks for the explanation!

**Review by DavidNix — APPROVED** [review]

**Pull request merged**